qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -192,6 +192,31 @@ class QasmSimulator(AerBackend):
       OpenMP parallelization. If parallel circuit or shot execution
       is enabled this will only use unallocated CPU cores up to
       max_parallel_threads (Default: 100).
+
+    These backend options only apply when using the ``"matrix_product_state"``
+    simulation method:
+
+    * ``"approximation_type"`` (int): Can be one of the following options:
+      NO_APPROX - means no approximation.
+      ABSOLUTE_APPROX - only the largest max_num_coefficients_for_approx
```

Comment:
I don't really understand what the different between these approximation types types are? It seems like we don't need this option, just two approximation options: a zero threshold, and a max bond dimension